cyclus/agents.pyx

```diff
@@ -1101,7 +1101,7 @@ cdef class _Region(_Agent):
         rtn = bool_to_py((<CyclusRegionShim*> (<_Agent> self).shim).DecendentOf(cpp_other))
         return rtn
 
-    def decomission(self):
+    def Decommission(self):
```

Comment:
This is an odd case change.... can’t really tell if it’s correct or an accident?  Did this not work before, at all?

Reply:
It was on purpose. It was to fix the spelling.

Reply:
I know the spelling was fixed, but I’m not clear on what the style guide says about the capitalization

Reply:
Ahh. Yes! That was a mistake. I was lazy and did a simple find replace. I'll go through and fix these up. Thanks!